rpm/xulrunner-qt5.spec

```diff
@@ -57,122 +57,99 @@ URL:        https://git.sailfishos.org/mer-core/gecko-dev
+Patch49:    0049-sailfishos-gecko-Restore-NotifyDidPaint-event-and-ti.patch
+Patch50:    0050-sailfishos-gecko-Adapt-build-configuration-for-Sailf.patch
+Patch51:    0051-sailfishos-webrtc-Update-GN-build-files-for-WebRTC.-.patch
+Patch52:    0052-sailfishos-gecko-Disable-desktop-sharing-feature-on-.patch
+Patch53:    0053-sailfishos-gecko-Enable-GMP-for-encoding-decoding.-J.patch
+Patch54:    0054-sailfishos-webrtc-Implement-video-capture-module.-JB.patch
+Patch55:    0055-sailfishos-webrtc-Regenerate-moz.build-files.-JB-537.patch
+Patch56:    0056-sailfishos-gecko-Drop-AudioPlayback-messages-if-no-e.patch
+Patch57:    0057-sailfishos-gecko-Get-ContentFrameMessageManager-via-.patch
+Patch58:    0058-sailfishos-gecko-Convert-panic-into-early-return-in-.patch
+Patch59:    0059-sailfishos-gecko-Allow-LoginManagerPrompter-to-find-.patch
+Patch60:    0060-sailfishos-gecko-Add-support-for-prefers-color-schem.patch
+Patch61:    0061-sailfishos-gecko-Update-hash-for-mapped_hyph.patch
+Patch62:    0062-sailfishos-gecko-Fix-content-action-integration-to-w.patch
+Patch63:    0063-sailfishos-gecko-Make-fullscreen-enabling-work-as-us.patch
+Patch64:    0064-sailfishos-gecko-Prioritize-GMP-plugins-over-all-oth.patch
+Patch65:    0065-sailfishos-gecko-Force-recycling-of-gmp-droid-instan.patch
+Patch66:    0066-sailfishos-gecko-Force-use-of-mobile-video-controls..patch
+Patch67:    0067-sailfishos-gecko-Fix-video-hardware-accelaration-not.patch
+Patch68:    0068-sailfishos-gecko-Add-a-video-decoder-based-on-gecko-.patch
+Patch69:    0069-sailfishos-gecko-Fix-audio-underruns-for-fullduplex-.patch
+Patch70:    0070-sailfishos-gecko-Bug-1750760-Create-ffmpeg59-module-.patch
+Patch71:    0071-sailfishos-gecko-Bug-1750760-Open-libavcodec.so.59-l.patch
+Patch72:    0072-sailfishos-gecko-Bug-1750760-Update-audio-and-video-.patch
+Patch73:    0073-sailfishos-gecko-Bug-1761471-FFmpeg-5.0-Get-frame-co.patch
+Patch74:    0074-sailfishos-gecko-Bug-1758948-FFmpeg-Use-AVFrame-pts-.patch
+Patch75:    0075-sailfishos-gecko-Ensure-audio-continues-when-screen-.patch
+Patch76:    0076-sailfishos-gecko-Fix-build-failure-due-to-rust-lang-.patch
+Patch77:    0077-sailfishos-gecko-Fix-unstable-name-collisions-warnin.patch
+Patch78:    0078-sailfishos-embedlite-egl-Fix-mesa-egl-display-and-bu.patch
+Patch79:    0079-sailfishos-gecko-Delete-startupCache-if-it-s-stale.patch
+Patch80:    0080-sailfishos-gecko-Hardcode-loopback-address-for-profi.patch
+Patch81:    0081-sailfishos-gecko-Start-using-user-agent-builder.-JB-.patch
+Patch82:    0082-sailfishos-gecko-Disallow-page-zooming-if-the-meta-v.patch
+Patch83:    0083-sailfishos-gecko-Add-preference-to-bypass-CORS-on-ns.patch
+Patch84:    0084-sailfishos-gecko-Get-12-24h-timeformat-setting-from-.patch
 
 BuildRequires:  rust
 BuildRequires:  rust-std-static
 BuildRequires:  cargo
 BuildRequires:  pkgconfig(Qt5Quick)
 BuildRequires:  pkgconfig(Qt5Network)
+BuildRequires:  pkgconfig(Qt5Widgets)
```

Comment:
Why do we need this?

Reply:
I'm not sure what the reason was, but the packages build fine without this so I've removed this line.

Reply:
There seems to be a requirement for that in gfx/cairo/cairo/src/cairo-qt-surface.cpp

Reply:
Thanks @mlehtima; I've removed the revert commit.